src/components/pages/profile/[name]/tabs/MoreTab/MoreTab.tsx

```diff
@@ -42,8 +45,8 @@ const MoreTab = ({ name, nameDetails, abilities }: Props) => {
       )}
       <Resolver
         name={name}
-        canEdit={abilities.canEdit}
-        canEditResolver={abilities.canEditResolver}
+        canEdit={!hasGlobalError && abilities.canEdit}
+        canEditResolver={!hasGlobalError && abilities.canEditResolver}
```

Comment:
afaik this isn't parity behaviour, not sure that this behaviour is ideal because the error could be unrelated

Reply:
depends on whether any of the nameDetails are fetched via a subgraph. or actually it doesn't matter - it just editrs a re